integration/src/main/kotlin/kotlinx/benchmark/integration/Runner.kt

```diff
@@ -1,20 +1,22 @@
 package kotlinx.benchmark.integration
 
-import kotlinx.benchmark.integration.GradleTestVersion.MinSupportedGradleVersion
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.GradleRunner
 import java.io.File
 
 class Runner(
     private val projectDir: File,
     private val print: Boolean,
-    private val gradleVersion: GradleTestVersion? = null,
+    gradleVersion: GradleTestVersion? = null,
 ) {
+    /** Defaults to the minimum Gradle version specified in [kotlinx.benchmark.gradle.BenchmarksPlugin] */
+    private val gradleVersion: GradleTestVersion = gradleVersion ?: GradleTestVersion.v7
```

Comment:
Could you please clarify why `v7("7.6.4")` is used instead of the `MinSupportedGradleVersion("7.4")`?

Reply:
it was a typo, thanks for spotting it